packages/web/config.ts

```diff
@@ -31,4 +31,12 @@ export const CONFIG = {
   googleDataApiKey: process.env.NEXT_PUBLIC_YOUTUBE_API_KEY,
   web3StorageToken: process.env.WEB3_STORAGE_TOKEN || '',
   openseaApiKey: process.env.OPENSEA_API_KEY || '',
+  onboardingGameDataURL:
+    'https://ipfs.fleek.co/ipfs/bafybeig5n6md7kemfjgdbguiepzisrt2y6icxhsj65kyrirs3kqgpv56eu',
+  openseaBaseURL: 'https://opensea.io',
+  polygonscanBaseURL: 'https://polygonscan.com',
```

Comment:
`polygonBaseURL` is already there `utils/networks` as "explorer" for polygon network.